src/journal/test-journal-interleaving.c

```diff
@@ -185,6 +185,15 @@ static void test_skip_one(void (*setup)(void)) {
         test_check_numbers_down(j, 9);
         sd_journal_close(j);
 
+        /* Seek to head twice, iterate down. */
+        assert_ret(sd_journal_open_directory(&j, t, 0));
+        assert_ret(sd_journal_seek_head(j));
+        assert_se(sd_journal_next(j) == 1);     /* pointing the first entry */
+        assert_ret(sd_journal_seek_head(j));
+        assert_se(sd_journal_next(j) == 1);     /* pointing the first entry */
```

Comment:
A preexisting condition, but "point the first entry" is not a valid sentence. "Pointing to the first entry" or "pointing at the first entry" or "before" or "after".

Reply:
Thanks. `to` is inserted.